packages/kit/src/template.ts

```diff
@@ -227,7 +224,9 @@ export async function _generateTypes (nuxt: Nuxt) {
   } satisfies TSConfig)
 
   const aliases: Record<string, string> = nuxt.options.alias
-
+  const EXTENSION_RE = /\b\.\w+$/g
+  // Exclude bridge alias types to support Volar
+  const excludedAlias = [/^@vue\/.*$/, /^#internal\/nuxt/]
```

Comment:
This would be a dangerous change as it introduces catastrophic backtracking with the signature syntax of (+)+ being a minimum polynomial greedy pattern.